frontend/src/lib/components/TeamInvitation.js

```diff
@@ -12,16 +13,29 @@ export function TeamInvitationLink({ user }) {
 }
 
 export function TeamInvitationContent({ user }) {
-    return (
-        <div>
-            <p>
-                <TeamInvitationLink user={user} />
-            </p>
-            Invite teammates with the link above.
-            <br />
-            Build an even better product, <i>together</i>.
-        </div>
-    )
+    const url = window.location.origin
+    const signup_token_state = user['team']['signup_token']
+    const signup_data = `{'team':{'signup_token': ${signup_token_state}}}`
```

Comment:
This should be just a JS object. Also, JSON strings are enclosed in **double** quotes (like "so"), so ultimately this couldn't have been decoded as JSON.

Reply:
Ah right, that makes sense. Thanks!